### DISCUSSION TITLE
Conf file overrides from command line when value is a comma separated list

### DISCUSSION THREAD

**Discussion by malloryprow** [dtcenter/METplus#2624] [Configuration] [answered]
I have a need to use the ability to override conf options on the command line. One of those is the PB2NC_MESSAGE_TYPE setting. I want to override it with multiple message types.

Setting in the conf file would normally look like `PB2NC_MESSAGE_TYPE=ADPUPA, AIRCAR, AIRCFT`.

On the command line I am doing `run_metplus.py -c SOME_PB2NC_CONF_FILE.conf -c config.PB2NC_MESSAGE_TYPE=ADPUPA,AIRCAR,AIRCFT`. Similarly, this needs to be done for `PB2NC_OBS_BUFR_VAR_LIST`.

I see in [run_metplus.py](https://github.com/dtcenter/METplus/blob/main_v5.1/ush/run_metplus.py) on line 107 it appears that this is not supported. Am I understanding that correct?

**Comment by georgemccabe** [comment, marked as answer]
Hi @malloryprow,

You are correct that list values are not supported on the command line. Issue #511 exists for this work, but it is not slated to be completed for this release. Does you need this functionality for this release?

**Reply by malloryprow** [reply]
No pressing needed to have this functionality in the release. I'm testing out using the instances functionality in place of this and then passing that on the command line. It worked out for the one case I needed it to, and now trying it on the other. Worst case if it doesn't work in the other one I can just support multiple conf files for what I need to do.

Thanks for confirming! I just wanted to make sure I wasn't missing anything.
